pkg/omf/functions/compat/refresh.fish

```diff
@@ -0,0 +1,14 @@
+function refresh -d "(deprecated) Refresh fish session by replacing current process"
+  printf 'warning: function %srefresh%s is deprecated and will be removed soon.\n %s\n' \
+  (set_color -u) (set_color normal) \
+  (status -t)[5] >&2
```

Comment:
i mean no matter how i call it, it only says "called on standard input" for me. even when that's clearly not the case. i feel like, given this, and given the inability to do anything with that information even if it was true, it makes the most sense to suppress "called on standard input". it's just not useful information :)